bin/main.rb

```diff
@@ -1 +1,123 @@
+    puts '-----------'
+    puts ' ' + @board[3].to_s + ' | ' + @board[4].to_s + ' | ' + @board[5].to_s + ' '
+    puts '-----------'
+    puts ' ' + @board[6].to_s + ' | ' + @board[7].to_s + ' | ' + @board[8].to_s + ' '
+  end
+
+  def draw_line
+    puts '############################################################'
+  end
+
+  def draw_welcome
+    puts '************************************************************'
+    puts '*                        TIC-TAC-TOE                       *'
+    puts '************************************************************'
+    puts "      by the Awesome Duo: Effie & David\n\n"
+    puts '                                                 '
+    puts '-------------GAME INSTRUCTIONS---------------'
+    puts 'The first player takes X and the second player takes O'
+    puts 'Choose numbers from 1 to 9 to select desired cell'
+    puts 'No duplicate numbers are allowed'
+    puts '--------------------------------'
+    puts 'Player 1: What is your name?:'
+    @player_one = gets.chomp
+    puts 'Player 2: What is your name?:'
+    @player_two = gets.chomp
+    puts '************************************************************'
+    puts "Player one X is #{player_one} and Player two O #{player_two}"
+    puts '************************************************************'
+    puts 'Game Start!'
+  end
+
+  def draw_player_turn(player)
+    puts '************************************************************'
+    puts "*                  #{player} its your turn!                *"
+    puts '************************************************************'
+  end
+
+  def draw_available_moves(array)
+    puts '************************************************************'
+    puts "*       This are the available moves  #{array}             *"
+    puts '************************************************************'
+  end
+
+  def invalid_move
+    puts '************************************************************'
+    puts '*                This move is not valid!                   *'
+    puts '************************************************************'
+  end
+
+  def winning_move
+    puts '************************************************************'
+    puts '*                This is a winning move!                   *'
+    puts '************************************************************'
+  end
+
+  def draw_move
+    puts '************************************************************'
+    puts '*                    This is a draw!                       *'
+    puts '************************************************************'
+  end
+end
+
+#########
+counter = 0
+# initial instructions for player
+game = DisplayInterface.new
+game.draw_welcome
+player_one = game.player_one
+player_two = game.player_two
+while counter < 9
+  # loop for each move
+  if game.get_current_player == player_one
+    game.draw_player_turn(player_one)
+    position = game.get_position.to_i
```

Comment:
There are missing code here
```ruby
game.draw_player_turn(player_one)
position = game.get_position.to_i

# check of invalid move
# if the move is valid game.take_place otherwise loop until the move is valid
```